### PR TITLE
PgMongo find, delete, insert

### PR DESCRIPTION
**Pull request by crazzysun**

### CODE REVIEW

File: src/main/java/pgmongo/PgMongoCollection.java [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class PgMongoCollection implements MongoCollection<Document> {
+    private java.sql.Connection connection;
+    String tableName;
```
Comment (ysarbaev) [inline src/main/java/pgmongo/PgMongoCollection.java]: private

File: src/main/java/pgmongo/PgMongoCollection.java [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+    }
+
+    @Override
+    public <TResult> FindIterable<TResult> find(Class<TResult> aClass) {
+        return null;
+    }
+
+    public FindIterable<Document> find(Bson query, Bson projection) {
+        try {
+            QueryTranslator qt = new QueryTranslator("json_data");
```
Comment (ysarbaev) [inline src/main/java/pgmongo/PgMongoCollection.java]: let's make "json_data" configurable

File: src/main/java/pgmongo/PgMongoCollection.java [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+    @Override
+    public void insertOne(Document Document) {
+        ArrayList<Document> tmpArr = new ArrayList<>();
+        tmpArr.add(Document);
+        insertMany(tmpArr);
+    }
+
+    @Override
+    public void insertMany(List<? extends Document> list) {
+        QueryTranslator qt = new QueryTranslator("json_data");
```
Comment (ysarbaev) [inline src/main/java/pgmongo/PgMongoCollection.java]: configurable

File: src/main/java/pgmongo/PgMongoCollection.java [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+    }
+
+    @Override
+    public DeleteResult deleteOne(Bson bson) {
+        return null;
+    }
+
+    @Override
+    public DeleteResult deleteMany(Bson bson) {
+        QueryTranslator qt = new QueryTranslator("json_data");
```
Comment (ysarbaev) [inline src/main/java/pgmongo/PgMongoCollection.java]: again

**Comment by crazzysun** [comment]
done

**Pull request merged**